flink-filesystems/flink-s3-fs-base/pom.xml

```diff
@@ -221,6 +221,36 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+
+		<!-- flink-runtime and curator dependencies are needed for the HAJobRunOnMinioS3StoreITCase -->
+		<dependency>
```

Comment:
I tried to introduce `<dependencyManagement/>` in `flink-s3-fs-base` instead. But they haven't been picked up by the submodules `flink-s3-fs-presto` and `flink-s3-fs-hadoop`. @zentol should this generally work and I just did something and should look into it once more? Or is it just not possible in this use case to utilize `<dependencyManagement/>`? 🤔